### DISCUSSION TITLE
Radiomaster  ER8GV

### DISCUSSION THREAD

**Discussion by miggy1952** [ExpressLRS/ExpressLRS#2366] [General]
Is it possible to change the sensor ID in the ER8GV?

In ETHOS the sensor ID fields are "Greyed Out" and not available to edit.

**Comment by pkendall64** [comment]
If we change it for ETHOS then we break EdgeTX. I think EdgeTX has a much bigger community. Perhaps a bug report for ETHOS is a better place to start.
